.pullapprove.yml

```diff
@@ -39,3 +39,18 @@ groups:
       enabled: true
       approve_regex: '^qa-passed'
       reject_regex: '^qa-failed'
+
+  docs-team:
+    conditions:
+      files:
+        - "documentation/*"
+        - "*.md"
+        - "*.rst"
+    required: 1
+    users:
+      - rcaballeromx
+      - mltullis
```

Comment:
And just to be clear - that is the github 01org organisation you need to be part of @mltullis - pm me if you need details.